static/src/javascripts/projects/common/modules/commercial/creatives/expandable-v3.js

```diff
@@ -0,0 +1,196 @@
+define([
+    'bean',
+    'bonzo',
+    'fastdom',
+    'common/utils/_',
+    'common/utils/$',
+    'common/utils/detect',
+    'common/utils/mediator',
+    'common/utils/storage',
+    'common/utils/template',
+    'common/views/svgs',
+    'text!common/views/commercial/creatives/expandable-v3.html'
+], function (
+    bean,
+    bonzo,
+    fastdom,
+    _,
+    $,
+	detect,
+    mediator,
+    storage,
+    template,
+    svgs,
+    expandableV3Tpl
+) {
+
+    /**
+     * https://www.google.com/dfp/59666047#delivery/CreateCreativeTemplate/creativeTemplateId=10028247
```

Comment:
Yes, this is referenced as expandableV3Tpl further down in the same file:

$expandableV3 = $.create(template(expandableV3Tpl, ...